data/json/furniture.json

```diff
@@ -1011,6 +1053,7 @@
         "id" : "f_bigmirror",
         "name": "standing mirror",
         "symbol": "{",
+        "description": "Look at yourself. Oh no, you're covered in gibs.",
```

Comment:
Odd phrasing. An imperative followed by a result. Maybe, `Looking goo- oh no, you're`...